anyio/_backends/_trio.py

```diff
@@ -2,12 +2,18 @@
 from types import TracebackType
 from typing import Callable, Optional, List, Type, Union
 
-import trio.hazmat
+try:
```

Comment:
Move the conditional imports after all others, as per PEP 8. I'm also not in favor of altering the `trio` package, so maybe instead alias the import as `lowlevel` or `trio_lowlevel`.